Kconfig.nrf

```diff
@@ -11,9 +11,23 @@ menu "Nordic nRF Connect"
 config BOARD_THINGY91_NRF9160NS
 	bool
 
-# nRF Connect SDK needs a larger default stacks for running certain tests.
+# Handle default to ensure that BT only will disable ENTROPY_CC3XX,
+# while multi protocol does not.
+config ENTROPY_CC3XX
+	default y if ZIGBEE || OPENTHREAD_NRF_SECURITY
```

Comment:
I think a better option would be to use `NET_L2_OPENTHREAD` instead of `OPENTHREAD_NRF_SECURITY`. The latter can be set to either `y` or `n` and still have functional OpenThread stack, while the fromer definitely enables/disables the stack.

Reply:
But does the NET_L2_OPENTHREAD imply usage of ENTROPY_CC3XX. From my understanding the need for OPENTHREAD_NRF_SECURITY is to override OPENTHREAD_MBEDTLS (built in mbed TLS)?

Reply:
Maybe my understanding is wrong, but doesn't setting the `ENTROPY_CC3XX` add the CC3XX generator as the default entropy source for all mbedTLS random number contexts, including both vanilla and nrf_security?

Reply:
In theory someone could configure OpenThread to use built in mbed TLS but entropy from CC3XX.

Reply:
Unfortunately mbed TLS from pure mbed TLS would be using a different context-size than the one used by the CryptoCell. In this case SW-implemented (by original mbed  TLS) would need to be used

This will also be more inefficient as the CTR_DRBG will be implemented by software rather than being HW accelerated

Do note that there are some limitations in the mechanism of selecting which driver that supports CTR_DRBG in the CSPRNG (it is a simple choice right now). We are working on a proposal to fix this to ensure that ENTROPY_CC3XX can be used in this case, but it would require adding a whole new "ENTROPY_HAS_CSPRNG_FEATURE" to enable (or not) in driver configuration. This would need upstream approval in Zephyr

Reply:
I suggest to just remove this redeclaration of `ENTROPY_CC3XX`. In `nrf/drivers/entropy/Kconfig` we already set this config to `default y`. Adding `default y if ZIGBEE || OPENTHREAD_NRF_SECURITY` has no effect